src/utils/rfq_blockchain_utils.ts

```diff
@@ -114,6 +115,35 @@ export class RfqBlockchainUtils {
         return this._exchangeProxy.executeMetaTransaction(metaTx, metaTxSig).getABIEncodedTransactionData();
     }
 
+    public async getNonceAsync(workerAddress: string): Promise<number> {
```

Comment:
These 4 functions below look like proxies, is there a need for this layer of abstraction?

Reply:
alternatively we could expose web3wrapper publicly

Reply:
this could be a little cleaner, even though less 00? up to you to decide :) - this current impl works fine too!